.ci/azure/linux.yml

```
@@ -555,6 +555,9 @@ jobs:
   - script: |
       set -e
       python3 -m pip install -r $(LAYER_TESTS_DIR)/requirements.txt
+      # Import 'test_utils' installed in '<package_test>/tests/python/openvino'
+      export LD_LIBRARY_PATH=$(python3 -m site --user-site)/openvino/libs:$LD_LIBRARY_PATH
+      export PYTHONPATH=${INSTALL_TEST_DIR}/python
```

```suggestion
      export PYTHONPATH=$(INSTALL_TEST_DIR)/python
```
For the sake of consistency.

---

it looks more confusing IMO.